TKeazirian.HTTPServer/Response/ResponseHeaderName.cs

```diff
@@ -0,0 +1,9 @@
+namespace TKeazirian.HTTPServer.Response;
+
+public static class ResponseHeaderName
+{
+    public const string Allow = "Allow";
+    public const string ContentType = "Content-Type";
+    public const string ContentLength = "Content-Length";
+    public const string Location = "Location";
+}
```

Comment:
I know in Java you could do this win an Enum; I am not sure if C# does it in the same way, but it may be worth looking into?

this is what it would look like in Java:
```java
public enum ResponseHeaderName
{
    Allow("Allow"),
    ContentType("Content-Type"),
    ContentLength("Content-Length"),
    Location("Location")
}

private final responseName;

ResponseHeaderName(String responseName)
{
    this.responseName = responseName;
}
```

Reply:
I say this only because classes are typically used for when you need multiple objects each of with would keep track of their own state.  Enums are more for buckets of values or custom types, which it feels like what you are trying to accomplish here.

Reply:
I did actually look into this! I saw some Java projects using `enum` this way.  I even tried it this Java way but it does not work.  Everywhere I have looked, all the docs say you cannot use string values in `enum`, only `int`.  

I found this [blog](https://josipmisko.com/posts/string-enums-in-c-sharp-everything-you-need-to-know) helpful.

There is a method `Enum.GetName()` that gets called like this:

```csharp
.SetHeaders(Enum.GetName(typeof(ResponseHeaderName), 0), _allowedMethods)
```

However, looking at that, you would have no idea which Header Name it's calling, because it uses the index as the second parameter.  Also, you cannot have `-` in `enum` so I would have to write some sort of method to add in a dash if the header name needed the dash (e.g. "Content-Type").  

I think this way seems to be the best option from the C# reading that I've done!

Reply:
gotcha; thanks for looking into it.